gsi.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <include name="base-q.xml"/>
+  <remote name="phh" fetch="https://github.com/phhusson/" />
+  <remote name="b2g-gsi" fetch="https://github.com/b2g-gsi/"/>
```

Comment:
this is stuff we should at least have a fork of and pull from, and it should be under `b2gos` (but I suspect you wanted to move forward quickly so you just went with this `b2g-gsi`)

Reply:
Right. To move fast, I created `b2g-gsi`.
According to your advice, we can folk repos from [kaiostech](https://github.com/kaiostech):
 - `platform_build`
 - `platform_system_vold`
 - `platform_system_core` 

and need to add sub-branch `gsi_gonk_android-10(?)` form `gonk_android-10.0.0_r11`
and then I PR patchs to each repos ?

Reply:
We would need to fork them, but I'd prefer if we could keep a single `gonk_android-10.0.0_r11` branch that works for both GSI and non GSI ?

Reply:
In the AOSP source code, the GSI branch is independent of the non-GSI branch, and I am not sure whether it can be merged. 
there is a compromise method: 
Move all patches to the `patcher` repository, including `platform_build`, `platform_system_vold`, etc. In this way we don't need additional fork repos. The disadvantage is that we need to update the `patcher` in time, if the relevant repos changes

Reply:
`patcher` was originally used to store patches for some stable and infrequently updated repos

Reply:
> In the AOSP source code, the GSI branch is independent of the non-GSI branch, and I am not sure whether it can be merged.

Ah, that's a game changer then.

Reply:
I tried to save all patches to `patcher`, and `gsi.xml` looks like [this](https://github.com/b2gOS/manifests/pull/5/files#diff-7650aae2e1139e89ccfea37113caf0e3d6427d4c96937a71fde7e5b2af91ecd7). We may just need to fork [`patcher` and  `device_phh_treble`](https://github.com/b2gOS/manifests/pull/5/files#diff-7650aae2e1139e89ccfea37113caf0e3d6427d4c96937a71fde7e5b2af91ecd7R15-R16)

Reply:
I tried to save all patches to `patcher`, and `gsi.xml` looks like [this](https://github.com/b2gOS/manifests/pull/5/files#diff-7650aae2e1139e89ccfea37113caf0e3d6427d4c96937a71fde7e5b2af91ecd). We may just need to fork [`patcher` and `device_phh_treble`](https://github.com/b2gOS/manifests/pull/5/files#diff-7650aae2e1139e89ccfea37113caf0e3d6427d4c96937a71fde7e5b2af91ecd7R15-R16)